### DISCUSSION TITLE
I haven’t received my education benefits.

### DISCUSSION THREAD

**Discussion by hakeemsultan1** [community/community#134680] [GitHub Education]
### Hi everyone,

Can you please activate my GitHub Education benefits? I'm eager to learn programming and take courses. Thank you!

**Comment by tariqodeh0** [comment]
Here’s how you can join:

### **Verify Your Student Status**
   - Go to the [GitHub Student Developer Pack](https://education.github.com/pack) page.
   - Click on the **Get your Pack** button.
   - You’ll be asked to verify your student status. GitHub will require proof that you are a student. You can do this by providing one of the following:
     - A school-issued email address.
     - A student ID card.
     - Any other document that verifies your enrollment at an educational institution.

### **Apply for the GitHub Student Developer Pack**
   - Once your student status is verified, you can apply for the GitHub Student Developer Pack, which includes free access to various development tools and resources.

Once you’re accepted, you’ll have access to a variety of tools and services designed to help you learn and develop your skills in coding and software development.

**Comment by prabhjotschugh** [comment]
Hi @hakeemsultan1  ,

The waiting time may vary depending on different factors, such as the volume of requests; it usually takes a few days, but sometimes it can take a little longer, even a couple of weeks;

When the benefits become available, the status label will turn purple and change to:

`Congrats! Your request for @hakeemsultan1  was approved and your benefits are now available.`

For further reference you can look at these discussions: [#132790](https://github.com/orgs/community/discussions/132790) , [#131665](https://github.com/orgs/community/discussions/131665), [#132470](https://github.com/orgs/community/discussions/132470) , [#133531](https://github.com/orgs/community/discussions/133531), [#134279](https://github.com/orgs/community/discussions/134279)

And also the [Student Developer Pack Application FAQs](https://github.com/orgs/community/discussions/111352) :

[How long after I submit my application will it be approved?](https://github.com/orgs/community/discussions/111352#how-long-after-i-submit-my-application-will-it-be-approved)
[How long after I’ve been approved will I receive my academic benefits?](https://github.com/orgs/community/discussions/111352#how-long-after-ive-been-approved-will-i-receive-my-academic-benefits)
